src/app/auto-suggest/auto-suggest.component.html

```diff
@@ -10,7 +10,10 @@
   <ng-container *ngFor="let suggestion of suggestions">
     <div class="suggestion" (click)="addLocation(suggestion.item)">
       {{ suggestion.item.location }}
-      <span class="source">
+      <span
+        class="source"
+        [style.color]="colorService.getColor(suggestion.item.source.name)"
```

Comment:
Two things I'm not super sure of here:

1. I vaguely have a sense that interpolating method calls into templates can easily go bad performance-wise. I don't know when exactly it should be avoided, though.
2. I never see services referenced in templates in the codebase I work in -- not sure if that's something to be avoided too.

Reply:
Yeah -- CDS isn't s source so much as an aggregation of sources. If you like we can look into tweaking the actual lambda script to make this info pass-through.

Reply:
Right! Not super important to me ATM though -- I have no particular need for this for now I think :) obvi def go ahead and add if it would be useful to you!